src/libraries/System.Private.CoreLib/src/System/Int32.cs

```diff
@@ -79,7 +79,7 @@ public override int GetHashCode()
 
         public override string ToString()
         {
-            return Number.FormatInt32(m_value, null, null);
+            return Number.Int32ToDecStr(m_value, digits: -1, provider: null);
         }
 
         public string ToString(string? format)
```

Comment:
Why not to inline these as well? Like the Int16 ones, etc.

Reply:
If you are worried about the duplication, it should be fine to let inliner take care of it. E.g. change the implementation of this `ToString` method to `public string ToString(string? format) => ToString(format, null);`